### ISSUE TITLE
Annotate file from Hi-C file

### ISSUE THREAD

**Issue by kvshams** [sldrcyang/nMAGMA#1]
Hi authors,
Congratulations for the article and nice work. How did you create/process the raw files for annotate files. It would be really helpful to include the data pre/post process scripts also in the repo.
Thanks,
Shams

**Comment by sldrcyang** [comment]
Dear Kvshams,

Thanks for your advice, we will consider to provide a more detailed script about the process. For creating Hi-C annotate files, we first annotated SNPs to REs based on their physical location using MAGMA directly, then assigned the SNPs to genes via SNP-to-RE annotations and RE-gene regulatory pairs (more details can be seen in Methods and Materials of nMAGMA paper).

Yang

**Comment by jamesdalg** [comment]
Yes. If you were able to, that would be wonderful. H-MAGMA doesn't provide scripts on how to process the data and doesn't really describe concretely how they processed their data. It's just frustrating because, even though it's a nature neuroscience paper, it's not reproducible and is largely useless except for the small amount of tissues they provide. If nMAGMA were to do this, it could increase your citations and h-index from users who can then use your work.

**Comment by sldrcyang** [comment]
Thanks for your advise. We have already provided a script named as Main_script.txt on how to conduct nMAGMA step by step. I hope that will be useful for you. If you have any question or suggestion, please do not hesitate to contact me.
